modules/apps/commerce/commerce-address-web/src/main/resources/META-INF/resources/country/details.jsp

```diff
@@ -42,21 +42,26 @@ CommerceRegionsStarter commerceRegionsStarter = commerceCountriesDisplayContext.
 	<aui:fieldset-group markupView="lexicon">
 		<aui:fieldset>
 			<aui:fieldset>
-				<aui:input autoFocus="<%= true %>" name="name" />
+				<liferay-ui:input-localized
+					autoFocus="<%= true %>"
+					cssClass="form-group"
+					name="name"
+					xml="<%= (country == null) ? StringPool.BLANK : country.getTitleMapAsXML() %>"
+				/>
 
 				<aui:input checked="<%= (country == null) ? false : country.getBillingAllowed() %>" inlineLabel="right" labelCssClass="simple-toggle-switch" name="billingAllowed" type="toggle-switch" />
 
 				<aui:input checked="<%= (country == null) ? false : country.getShippingAllowed() %>" inlineLabel="right" labelCssClass="simple-toggle-switch" name="shippingAllowed" type="toggle-switch" />
 
-				<aui:input label="two-letter-iso-code" name="twoLettersISOCode" />
+				<aui:input field="a2" fieldParam="twoLettersISOCode" label="two-letter-iso-code" name="twoLettersISOCode" />
```

Comment:
Same suggestion here as with the other parameters. It may ultimately be easier to use the proper parameter names.